mahmoudghorbelMG/terraform-provider-azurermagw
# Dev (#5)

* first draft backend http settings

* probe problem resolved

* remove main.go prob

* nil probe

* probe state

* probe state correct

* check element name

* nil probe

* jh

* {Null: true}

* optional param set to null

* value false for pick

* //

* probe name true/false

* PickHostNameFromBackendAddress is jason

* PickHostNameFromBackendAddress become required

* AffinityCookieName ""

* Unknown: true

* Computed: true,

* PlanModifiers:

* rectif

* print

* json response

* AffinityCookieName default value

* rectify CookieBasedAffinity

* DEFAULT affinity_cookie_name

* ZERZER

* &&(str.Value !="")

* pick_host_name_from_backend_address default value

* first draft with probe

* correct probe null

* probe_json.Properties.Match = nil

* print update probe_plan

* test

* h

* check

* req.Plan.Get

* match as types.Object

* make match param required

* intPlanModifiers

* intDefaultModifier

* AttributeConfig

* probe name verification

* reorganise update method

* first draft http listener

* set instead list for http listeners

* map instead set

* set

* list

* k

* g

* update http listener

* gh

* print http listener

* s

* ok

* cert

* var

* condition

* print

* print

* hostname ok

* cert

* create ok

* hostname null prob

* nul true

* fatal-missing hostname

* set instead of list

* h

* new http listener

* probe without error

* with ssl certif for http

* df

* sd

* log

* go mod tidy

* c

* zz

* & *

* if hasField

* Required

* d

* print

* azer

* & *

* s

* no &

* else{}

* bb

* plan.Http_listener

* nil in read

* nil nil

* check state

* remove hhtp listener in update

* first draft ssl certificate

* rectify ssl name

* s

* check

* redirect conf not complete

* complete redirection conf

* first draft request routing rule

* priorities length

* j

* v

* check before create

* return

* update

* check ssl certif

* check redirect

* check https http

* result WebappBinding

* fg

* first full draft

* fix

* fix

* print

* fix2

* priority string fix

* last fix priority

* cleaner

* clean

* add release.yml and json file

* rm examples

* Delete examples directory

* test

* citeo.com/devops/azurermagw

* clean

Co-authored-by: Mahmoud Ghorbel <mahmoud.ghorbel.p@citeo.com>
